src/executors/IntervalExecutor.js

```diff
@@ -24,6 +48,12 @@ class IntervalExecutor extends ExecutorAbstract {
     this._interval();
   }
 
+  /**
+   * Internal recursive loop function.  Runs executor every (this.milliseconds) provided
+   * that the condition still evaluates truthy and (this.maxLoops) is not exceeded
```

Comment:
'the condition callback' to be explicit. Let's not wrap code references in parenthesis but in backticks per Markdown. I'm pretty sure most JS documentation engines recognize backticks as code wraps.